packages/agent/conf/demos/dbc/oid4vci_metadata/dbc.json

```diff
@@ -59,39 +59,39 @@
           }
         ],
         "order": [
-          "Voornaam",
-          "Achternaam",
+          "firstName",
+          "lastName",
           "email"
         ],
         "credentialSubject": {
-          "Voornaam": {
+          "firstName": {
             "value_type": "string",
             "display": [
               {
-                "name": "FirstName"
+                "name": "First Name"
```

Comment:
this is the default branding when there is no locale. it is a dutch conference. should we not set this to dutch?